javascript_nodejs/2.echobot-with-counter/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "echobot-with-counter",
+  "version": "1.0.0",
+  "description": "Bot Builder v4 echo bot sample",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha tests/*.test.js",
```

Comment:
Are we having tests for each sample/are we having a specific sample that shows how to test using transcripts? 

Does this line need to be removed?